packages/vapor/scss/tables/table.scss

```diff
@@ -137,7 +137,10 @@
     td {
         padding: $table-row-top-bottom-padding $table-cell-left-right-padding;
         text-align: left;
-        border-bottom: $table-border;
+
+        &:not(.no-border-bottom) {
```

Comment:
We already have the `mod-no-border-bottom` class.
https://github.com/coveo/react-vapor/blob/4d62e60a5a1c2e66fa7326f87f0619e535baad15/packages/vapor/scss/utility/border.scss#L23
Can it be used instead ?

Reply:
![image](https://user-images.githubusercontent.com/52677246/80491809-32fbff80-8931-11ea-8a99-c16c96f6dd9f.png)
It seems to be overrode because of the precedence :thinking:

Reply:
:+1: Then perhaps the css rule could be
```suggestion
        &:not(.mod-no-border-bottom) {
```
This way we won't have to use another new class (`no-border-bottom`)

Reply:
Yes!